### ISSUE TITLE
Invalid import with latest version

### ISSUE THREAD

**Issue by carl0shd** [segmentio/analytics-ios#1020]
Hi, I am using the latest Analytics pod version `4.1.6`, and when I try to import the library in the AppDelegate, Xcode says:  `'Segment/SEGAnalytics.h' file not found`.

Happens the same thing with the Leanplum integration.
<img width="1354" alt="Screen Shot 2021-10-26 at 12 08 50 PM" src="https://user-images.githubusercontent.com/8359234/138927407-3d734729-c24f-4f22-a37e-74fe595eec10.png">

I run `pod install`, clean the build, and nothings work. I am using Xcode 13, I don't know if this is related https://github.com/segmentio/analytics-ios/pull/994.

A fun fact is that if I use `#import <Analytics/SEGAnalytics.h>` instead of `#import <Segment/SEGAnalytics.h>` the app compiles and works perfectly, but I still have the same problem with the Leanplum library, since it uses `#import <Segment/SEGAnalytics.h>`

PD: is the same problem with https://github.com/segmentio/analytics-ios/issues/1007

**Comment by sirajtahra** [comment]
+1

**Comment by sirajtahra** [comment]
@carl0shd were you able to fix the issue ?

**Comment by carl0shd** [comment]
> @carl0shd were you able to fix the issue ?

Yes, simply do a fork of the Leanplum SDK library, and change the default import to `#import <Segment/SEGAnalytics.h>` because appears segment will not update the SDK 😥.
